package main

func main() {
	//create an error
	// panic stops the application
	panic("this error is created by me")

}
